src/orderbook.js

```diff
@@ -125,8 +128,12 @@ function OrderBook(remote,
     });
   }
 
-  function updateFundedAmountsWrapper(transaction) {
-    self.updateFundedAmounts(transaction);
+  function onTransactionWrapper(transaction) {
```

Comment:
in this case I must save result of `self.onTransaction.bind(self)` into var, because I need to do `self._remote.removeListener('transaction', onTransactionWrapper);`